configs/next/packages/mpfr/sources

```diff
@@ -20,8 +20,8 @@
 #
 
 ATSRC_PACKAGE_NAME="GNU MPFR Library"
-ATSRC_PACKAGE_VER=3.1.7
-ATSRC_PACKAGE_REV=11775
+ATSRC_PACKAGE_VER=4.1
```

Comment:
@gftg85 I prefer to use 4.1 instead of 4.1.0-dev because as we are following the master branch the point release can change often.